src/renderer/components/ErrorModal/index.tsx

```diff
@@ -0,0 +1,75 @@
+import React, { useState } from 'react';
+import Store from "electron-store";
+import { setupInstallPath, setupLiveriesPath } from 'renderer/actions/install-path.utils';
+import { reloadLiveries } from '../AircraftSection/LiveryConversion';
+
+const settings = new Store();
+
+export const ErrorModal = (): JSX.Element => {
```

Comment:
Rename file as well

```suggestion
export const PathErrorModal = (): JSX.Element => {
```

Reply:
This was intended to be used for other errors as well in future. That's why I have a content function